src/logic/operate.js

```diff
@@ -0,0 +1,29 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const firstNumber = new Big(numberOne);
+  const secondNumber = new Big(numberOne);
```

Comment:
Kindly change "numberOne" to "numberTwo", currently we have 2 of the "numberOne". Let us fix it